javascript-source/games/gambling.js

```diff
@@ -102,7 +102,7 @@
 		 * @commandpath gamblesetwinningrange [range] - Set the winning range from 0-100.
 		 */
 		if (command.equalsIgnoreCase('gamblesetwinningrange')) {
-			if (!action || action.includes('-')) {
+			if ((!action) || isNaN(action) || action.includes('-') || (action < 1) || (action > 100)) {
```

Comment:
You don't need to add everything in specific parentheses, also since you're updating this change `!action` to `action === undefined`, and use `isNaN(parseInt(action))` something rhino will mess that up.

e.g `((!action) || (action < 1))`